docs/scalardb-supported-databases.md

```diff
@@ -1,31 +1,141 @@
-| ScalarDB 3.6 |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  |
-| ScalarDB 3.5 |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  |
-| ScalarDB 3.4 |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  |
-| ScalarDB 3.3 |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  |
-| ScalarDB 3.2 |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  |
-| ScalarDB 3.1 |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  |
-| ScalarDB 3.0 |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  Not tested  |  Not tested  |  Not tested  |  ✅  |  ✅  |  ✅  |  Not tested  |  Not tested  |  ✅  |  ✅  |  |
-| ScalarDB 2.4 |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  |
-| ScalarDB 2.3 |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  |
-| ScalarDB 2.2 |  ✅  |  ✅  |  ✅  |  ✅  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  |
-| ScalarDB 2.1 |  ✅  |  ✅  |  ✅  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  |
-| ScalarDB 2.0 |  ✅  |  ✅  |  ✅  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  |
+# Supported Databases
+
+ScalarDB supports the following databases and their versions.
+
+## Amazon DynamoDB
+
+| Version | Amazon DynamoDB |
+|:------------------|:---------------|
+| **ScalarDB 3.10** | ✅ |
+| **ScalarDB 3.9** | ✅ |
+| **ScalarDB 3.8** | ✅ |
+| **ScalarDB 3.7** | ✅ |
+| **ScalarDB 3.6** | ✅ |
+| **ScalarDB 3.5** | ✅ |
+| **ScalarDB 3.4** | ✅ |
+
+## Apache Cassandra
+
+{% capture notice--info %}
+**Note**
+
+For requirements when using Cassandra or Cassandra-compatible databases, see [Cassandra or Cassandra-compatible database requirements](requirements/#cassandra-or-cassandra-compatible-database-requirements).
+{% endcapture %}
+
+<div class="notice--info">{{ notice--info | markdownify }}</div>
+
+| Version | Apache Cassandra 4.1 | Apache Cassandra 4.0 | Apache Cassandra 3.11 | Apache Cassandra 3.0 | Apache Cassandra 2.2 |
+|:------------------|:---------------|:-----------------|:----------------|
+| **ScalarDB 3.10** |  |  | ✅ | ✅ | ✅ |
+| **ScalarDB 3.9** |  |  | ✅ | ✅ | ✅ |
+| **ScalarDB 3.8** |  |  | ✅ | ✅ | ✅ |
+| **ScalarDB 3.7** |  |  | ✅ | ✅ | ✅ |
+| **ScalarDB 3.6** |  |  | ✅ | ✅ | ✅ |
+| **ScalarDB 3.5** |  |  | ✅ | ✅ | ✅ |
+| **ScalarDB 3.4** |  |  | ✅ | ✅ | ✅ |
```

Comment:
I see a broken table for this:
https://github.com/scalar-labs/scalardb/blob/fca2294971edcfda26868188798a03ac1623e990/docs/scalardb-supported-databases.md#apache-cassandra

Can we fix it?

Reply:
Ah yes, I had added columns for Cassandra 4.0 and 4.1 and forgot to add extra extra bars after the header row. Show be fixed now.

Interesting that the docs site rendered the tables but GitHub didn't.🤔